### PR TITLE
i18n(fr): Update `getting-started.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? Give us a brief description.
Updated `getting-started.mdx` based on these 2 commits f6a690d0842b679090b649d794859722b32cee28 973d1076bf63b0e9fb5418efe524ccdde118a880

### CODE REVIEW

File: docs/src/content/docs/fr/getting-started.mdx [hunk 1]
```diff
@@ -81,6 +81,43 @@ Une fois que vous avez créé et personnalisé votre site Web Starlight, vous po
 
 [Pour en savoir plus sur le déploiement d'un site Astro, consultez la documentation Astro](https://docs.astro.build/fr/guides/deploy/)
 
+## Updating Starlight
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/getting-started.mdx:84]:
```suggestion
## Mettre à jour Starlight
```

File: docs/src/content/docs/fr/getting-started.mdx [hunk 2]
```diff
@@ -89,4 +126,4 @@ Utilisez la liste croissante de guides dans la barre latérale pour vous aider 
 
 Si vous ne trouvez pas votre réponse dans cette documentation, veuillez consulter la [documentation complète d'Astro](https://docs.astro.build). Votre question peut être résolue en comprenant comment Astro fonctionne en général, sous ce thème Starlight.
 
-Vous pouvez également obtenir de l'aide dans le [Discord Astro](https://astro.build/chat/) de notre communauté active et sympathique ! Vous pouvez poster des questions dans notre forum `#support` ou visiter notre canal dédié `#starlight` pour discuter des développements en cours et plus encore !
+Vous pouvez également vérifier s'il existe des [problèmes connus de Starlight sur GitHub](https://github.com/withastro/starlight/issues) et obtenir de l'aide dans le [Discord Astro](https://astro.build/chat/) de notre communauté active et sympathique ! Vous pouvez poster des questions dans notre forum `#support` avec le tag "starlight" ou visiter notre canal dédié `#starlight` pour discuter des développements en cours et plus encore !
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/getting-started.mdx:129]: Reference for the "post" translation https://www.culture.fr/franceterme/terme/INFO738

```suggestion
Vous pouvez également vérifier [tous les problèmes connus de Starlight sur GitHub](https://github.com/withastro/starlight/issues) et obtenir de l'aide dans le [Discord Astro](https://astro.build/chat/) de notre communauté active et sympathique ! Publiez des questions dans notre forum `#support` avec le tag "starlight" ou visiter notre canal dédié `#starlight` pour discuter des développements en cours et plus encore !
```

**Review by HiDeoo — COMMENTED** [review]
Neat. Left a suggestion for a missing title and a slight change to the last sentence but it's looking good.

**Comment by dreyfus92** [comment]
Thank you for the review @HiDeoo 😁

**Review by delucis — APPROVED** [review]
Thank you for the update @dreyfus92 and for reviewing @HiDeoo 🥳

**Pull request merged**